Assets/HoloToolkit/UX/Scripts/Buttons/Button.cs

```diff
@@ -93,29 +97,55 @@ public bool RequireGaze
         /// <summary>
         /// Last state of hands being visible
         /// </summary>
-        private bool _bLastHandVisible = false;
+        private bool lastHandVisible = false;
```

Comment:
Just keep in mind that renaming these will break the previous serialization people have in their projects.

Reply:
[There is an attribute we should be using to make sure it doesn't break](https://docs.unity3d.com/ScriptReference/Serialization.FormerlySerializedAsAttribute.html).
```
[FormerlySerializedAs("myValue")]
private string myValue;
```

Reply:
What serialization does this break? None of the names I changed have the `[SerializeField]` tag, or are you referring to something else?

Reply:
Good to know about that `[FormerlySerializedAs]` tag! That seems very useful.

Reply:
Oh, I thought some were public before, then it's no worries.